examples/real_data.ipynb

```diff
@@ -0,0 +1,769 @@
+    "obs_galaxies = obs_galaxies[select]\n",
+    "\n",
+    "obs_galaxies['id'] = np.arange(len(obs_galaxies))\n",
+    "\n",
+    "# Put galaxy values on arrays.\n",
+    "gal_ra = obs_galaxies['ra'] # Galaxies Ra in deg\n",
+    "gal_dec = obs_galaxies['dec'] # Galaxies Dec in deg\n",
+    "gal_e1 = obs_galaxies['e1'] # Galaxies elipticipy 1\n",
+    "gal_e2 = obs_galaxies['e2'] # Galaxies elipticipy 2\n",
+    "gal_z = obs_galaxies['z'] # Galaxies observed redshift\n",
+    "gal_id = obs_galaxies['id'] # Galaxies ID"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3c1f44ef-ee3b-490b-8d54-4d433b50a93f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Shear profile\n",
+    "# Using/testing clmm.dataops functions (details are in the notebook for Paper V1.0).\n",
+    "# This step is for testing the CLMM functionalities; we will not use the result in later step.\n",
+    "\n",
+    "import clmm.dataops as da\n",
+    "# Convert elipticities into shears.\n",
+    "gal_ang_dist, gal_gt, gal_gx = da.compute_tangential_and_cross_components(cluster_ra, cluster_dec,\n",
+    "                                                                          gal_ra, gal_dec,\n",
+    "                                                                          gal_e1, gal_e2,\n",
+    "                                                                          geometry=\"flat\")\n",
+    "\n",
+    "# Measure profile.\n",
+    "field_size = 6 # Mpc box size\n",
+    "\n",
+    "profile = da.make_radial_profile([gal_gt, gal_gx, gal_z],\n",
+    "                                 gal_ang_dist, \"radians\", \"Mpc\",\n",
+    "                                 bins=da.make_bins(0.01, field_size/2., 50),\n",
+    "                                 cosmo=cosmo,\n",
+    "                                 z_lens=cluster_z,\n",
+    "                                 include_empty_bins=False)\n",
+    "print(f'Profile table has columns: {\", \".join(profile.colnames)},')\n",
+    "print('where p_(0, 1, 2) = (gt, gx, z)')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "42573e50-85cc-4977-baf4-723711792c6b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Using the GalaxyCluster object.\n",
+    "\n",
+    "import clmm\n",
+    "from clmm.utils import convert_units\n",
+    "\n",
+    "# Create a GCData with the galaxies.\n",
+    "galaxies = clmm.GCData([gal_ra, gal_dec, gal_e1, gal_e2, gal_z, gal_id],\n",
+    "                      names=['ra', 'dec', 'e1', 'e2', 'z', 'id'])\n",
+    "\n",
+    "# Create a GalaxyCluster.\n",
+    "cluster = clmm.GalaxyCluster(\"Name of cluster\", cluster_ra, cluster_dec,\n",
+    "                                   cluster_z, galaxies)\n",
+    "\n",
+    "# Convert elipticities into shears for the members.\n",
+    "cluster.compute_tangential_and_cross_components(geometry=\"flat\")\n",
+    "print(cluster.galcat.colnames)\n",
```

Comment:
Please, remove `geometry='flat'`. The default is now to use the correct `curve` geometry, using astropy functionality.

Reply:
Done.